src/main/res/values/strings.xml

```diff
@@ -139,23 +139,23 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
+  <string name="update_notifications">Notifications about updates</string>
```

Comment:
I'm still concerned about the length of this string. Are there any problems "Notify about updates"?
(Hope, this is the last review on this PR!)

Reply:
It isn't great, but there aren't any options that meet the requirements that I can find.
"Notify me of updates" was the closest I got.